spec/octokit/client/hooks_spec.rb

```diff
@@ -131,4 +131,13 @@
       end
     end # .remove_org_hook
   end # with org hook
+
+  describe ".parse_payload", :vcr do
+    it "parses payload string" do
+      test_json = fixture('create_payload.json').read
+      test_answer = fixture('create_payload_hash.txt').read
```

Comment:
This approach might be overkill since the format of `inspect` could change if test dependencies are added to the project. I think we can just assert a couple of the fields at a couple of depths on `result` and be just fine.